lib/lob/client.ex

```diff
@@ -53,7 +53,7 @@ defmodule Lob.Client do
   end
 
   def process_response_body(body) do
-    Parser.parse!(body, keys: :atoms)
+    Parser.parse!(body, %{keys: :atoms})
```

Comment:
It's more idiomatic to pass options as a keyword list rather than a map.  I think I hit a snag in my recent updates of other Elixir libraries where I had to modify the format of the options from a map to the keyword list format.

Secondly, I think this would be cleaner if it called `Poison.decode/2` instead of `Poison.Parser.parse/2`.  Poison routes execution over to its parser module, but that is an implementation detail that we don't necessarily want to rely on.   Also, that would keep things a bit more flexible with respect to which specific JSON package in use (nowadays, `Jason` is more prevalent than `Poison`).  E.g. 

```
# Set up a named alias
alias Poison, as: Json

# ...


  def process_response_body(body) do
    Json.decode!(body, keys: :atoms)
  end
```

This way, if you ever decided to use `Jason` instead of `Poison`, you'd just need to make a single change at the top of the file (or you could even move this into configuration).  But here you see the importance of using the keyword list instead of a map -- although `Poison` may tolerate (or tolerated?) options provided as maps, it's less likely that other packages would do the same.

Just food for thought!